src/SharedCode/EnvironmentHelper.cs

```diff
@@ -698,6 +698,27 @@ public static bool StartUpdate()
             }
         }
 
+        public static bool StartManager()
+        {
+            try
+            {
+                var gameRoot = Path.GetFullPath(GameRootDirectory).TrimEnd('\\', '/', ' ');
+                var updaterPath = Path.Combine(_kkmanagerDirectory, "KKManager.exe");
+
+                if (!File.Exists(updaterPath))
+                    throw new FileNotFoundException("Coult not find the KKManager", updaterPath);
+
+                var args = $"\"{gameRoot}\" {_updateSourcesOverride}";
+
+                return StartProcess(new ProcessStartInfo(updaterPath) { WorkingDirectory = $@"{_kkmanagerDirectory}", Arguments = args }) != null;
```

Comment:
```cs
WorkingDirectory = $@"{_kkmanagerDirectory}"
```
could just be
```cs
WorkingDirectory = _kkmanagerDirectory
```

Reply:
Fixed it

Reply:
Same can be done on line 692 for updater method.